misc/python/materialize/cli/run.py

```diff
@@ -96,6 +109,11 @@ def main() -> int:
         help="Only build, don't run",
         action="store_true",
     )
+    parser.add_argument(
```

Comment:
Should we enable this by default? Does it cause any slowness during build, dialogs that need to be clicked? If enabling this all the time for mac users is invisible to them, seems ok to always enable.

Reply:
It does not cause any slowness, nor opens any dialogs, so enabling it by default sounds good to me!